### DISCUSSION TITLE
ПОМОГИТЕ

### DISCUSSION THREAD

**Discussion by Freakssd** [aiogram/aiogram#1214] [General]
Здравствуйте.  Нужна помощь, делаю большого бота и разбил его на файлы и когда я регистрирую callback хэндлеры у меня какое то ограничение по регистрации хэндлеров, то есть больше какого то количества хэндлеров я не могу зарегестрировать. Подскажите что с этим делать.

---
> **Google translate:**
> 
> Hello. I need help, I'm making a big bot and split it into files, and when I register callback handlers, I have some kind of restriction on registering handlers, that is, I cannot register more than a certain number of handlers. Tell me what to do with it.

**Comment by JrooTJunior** [comment]
Don't use any language than English here.

What kind of error do you have?

Can you provide minimum reproducible example and traceback?
